bindata/network/multus-admission-controller/001-service.yaml

```diff
@@ -13,7 +13,9 @@ metadata:
 {{- if .HyperShiftEnabled}}
     network.operator.openshift.io/cluster-name: {{.ManagementClusterName}}
 {{- end }}
+{{- if not .HyperShiftEnabled or eq .CAConfigMap ""}}
```

Comment:
This fails in CI:
```
I0312 20:47:44.272052       1 log.go:245] Failed to render: failed to render multus admission controller manifests: failed to render file bindata/network/multus-admission-controller/001-service.yaml: failed to render manifest bindata/network/multus-admission-controller/001-service.yaml: template: bindata/network/multus-admission-controller/001-service.yaml:16:7: executing "bindata/network/multus-admission-controller/001-service.yaml" at <not>: wrong number of args for not: want 1 got 5
```

Is it possible for `CAConfigMap` to be empty in HyperShift? it doesn't look like it from the code logic and there is no check in [multus-admission-controller/monitor.yaml](https://github.com/openshift/cluster-network-operator/pull/2309/files#diff-8c63f39010003c98df7342fc123f4fd07c270a9b0a7369f341064350207b1aaf)

Please add a render unit test somewhere in `pkg/network/multus_admission_controller_test.go`

Reply:
For older deployments, it's possible it would be empty. I will add a check in [multus-admission-controller/monitor.yaml](https://github.com/openshift/cluster-network-operator/pull/2309/files#diff-8c63f39010003c98df7342fc123f4fd07c270a9b0a7369f341064350207b1aaf).

Reply:
`CAConfigMap` is set to the `CA_CONFIG_MAP` env if it exists or `openshift-service-ca.crt` otherwise. Are you saying that there is a scenario in which `CA_CONFIG_MAP` env is set to an empty value? If so please add a comment.

Reply:
Added comments.